### PR TITLE
Fix format of note on max_results_per_page and emphasize

### PR DESCRIPTION
**Pull request by Fryguy**
@chessbyte Please review.

cc @dehawkins512

### CODE REVIEW

File: api/overview/pagination.md [hunk 1]
```diff
@@ -72,26 +72,18 @@ following:
...
-
-About max\_results\_per\_page
-
-</div>
-
-If the default value for max\_results\_per\_page does not meet your
+If the default value for max_results_per_page does not meet your
 needs, it can be changed using the Advanced Settings feature in the UI.
-It is recommended however that it is not set to some arbitrarily large
+However, we *DO NOT* recommended setting it to some arbitrarily large
```
Comment (chessbyte) [inline api/overview/pagination.md:85]: typo: `recommended` should be `recommend`

Comment (chessbyte) [reply api/overview/pagination.md:85]: also, why the `we`?  maybe better wording would be: `However, it is *NOT* recommended to set it to ...`

Comment (Fryguy) [reply api/overview/pagination.md:85]: I originally had it that way and it sounded weird, but now that I'm reading it the way you wrote it, it sounds fine...will change.

**Comment by Fryguy** [comment]
@chessbyte Updated.

**Comment by Fryguy** [comment]
jansa backport details

```text
$ git log -1
commit 54c3e401d1e2f282a13643c3e35b45821280d022
Author: Oleg Barenboim <chessbyte@gmail.com>
Date:   Thu Jul 30 16:19:42 2020 -0400

    Merge pull request #1470 from Fryguy/fix_note

    Fix format of note on max_results_per_page and emphasize

    (cherry picked from commit 6dffbf7f1238987efba185feb8aaa39ef18532f7)

```

**Pull request merged**